### PR TITLE
Build with rollup

### PR DESCRIPTION
**Pull request by JGiter**
### Description

Replacing bili with rollup, because bili is not maintained anymore, which prevents from fixing

### Contributor checklist

- [x] Breaking changes - check for any existing interfaces changes that are not backward compatible, removed method etc.
- [x] Documentation - document your code, add comments for method, remember to check if auto generated docs were updated.
- [x] Tests - add new or updated existed test for changes you made.
- [x] Migration guide - add migration guide for every breaking change.
- [x] Configuration correctness - check that any configuration changes are correct ex. default URLs, chain ids, smart contract verification on [Volta explorer](https://volta-explorer.energyweb.org/) or [EWC explorer](https://explorer.energyweb.org/).

### CODE REVIEW

File: tsconfig.test.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["e2e/**/*"],
+  "ts-node": {
+    "compilerOptions": { "module": "CommonJS" }
+  }
```
Comment (jrhender) [inline tsconfig.test.json:6]: Is this needed? The root `tsconfig.json` from which this config extends already has these `ts-node` options, I believe.

File: tsconfig.build.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["src/**/*", "ethers/**/*"],
+  "ts-node": {
+    "compilerOptions": { "module": "CommonJS" }
+  }
```
Comment (jrhender) [inline tsconfig.build.json:6]: Is this needed? The root `tsconfig.json` from which this config extends already has these `ts-node` options, I believe.

File: package.json [hunk 3]
```diff
@@ -64,18 +63,18 @@
...
-    "@energyweb/vc-verification": "2.2.1-alpha.313.0",
+    "@energyweb/vc-verification": "2.2.1-alpha.314.0",
     "@ensdomains/ens": "^0.6.2",
     "@ew-did-registry/claims": "0.8.1-alpha.1037.0",
     "@ew-did-registry/credentials-interface": "0.8.1-alpha.1037.0",
     "@ew-did-registry/did": "0.8.1-alpha.1037.0",
     "@ew-did-registry/did-document": "0.8.1-alpha.1037.0",
     "@ew-did-registry/did-ethr-resolver": "0.8.1-alpha.1037.0",
-    "@ew-did-registry/did-ipfs-store": "0.7.1-alpha.816.0",
+    "@ew-did-registry/did-ipfs-store": "^0.7.1-alpha.816.0",
```
Comment (jrhender) [inline package.json:77]: Why the addition of the caret here? I think a pinned version is clearly and maybe a bit safer (and the other 
`@ew-did-registry` packages have a pinned version)

File: package.json [hunk 4]
```diff
@@ -2,20 +2,19 @@
...
+  "types": "lib/src/index.d.ts",
   "files": [
-    "dist"
+    "lib",
+    "lib.esm"
   ],
   "scripts": {
     "lint": "eslint ./src/**/*.ts ./e2e/**/*.ts",
-    "prebuild": "rimraf dist ethers",
-    "build": "npm run build:contracts && npm run build:ts && copyfiles \"ethers/**/*.d.ts\" dist",
```
Comment (jrhender) [inline package.json:14]: Is `copyfiles \"ethers/**/*.d.ts\" dist` no longer needed? If it is not needed anymore, perhaps `copyfiles` can be removed as a dev dependency

**Review by jrhender — COMMENTED** [review]
A few comments/questions but looks like a good change 👍 . Thanks for your work on this @JGiter and apologies for the slow PR review

File: tsconfig.build.json (hunk 2 above)
Comment (JGiter) [reply tsconfig.build.json:6]: yes, it can be removed :+1:

File: package.json (hunk 4 above)
Comment (JGiter) [reply package.json:14]: yes, contract definitions are part of the code, so they are included in build. I will remove `copyfiles`. good catch. may be we can use https://www.npmjs.com/package/npm-check

**Review by nichonien — APPROVED** [review]

**Review by jrhender — APPROVED** [review]
Looks good to me, thanks @JGiter

**Pull request merged**